Samples/kmp-app/shared/src/androidMain/kotlin/sample/kpm_app/Proxy.kt

```diff
@@ -0,0 +1,23 @@
+package sample.kpm_app
+
+import android.content.Context
+import io.sentry.kotlin.multiplatform.ContextProvider
+import io.sentry.kotlin.multiplatform.Sentry
+
+class Proxy {
+    /**
+     * For manual initialization
+     */
+    fun init(dsn: String, resolveContext: ()-> Context) {
+        ContextProvider.init(resolveContext)
```

Comment:
In case auto-init is disabled in AndroidManifest.xml, this will set the Application context needed for SentryAndroid.init(..): https://github.com/buenaflor/sentry-kotlin-multiplatform/blob/main/src/androidMain/kotlin/io/sentry/kotlin/multiplatform/SentryAndroid.kt

Reply:
So we need to be sure that the final users don't need to call this method, but rather we do that in the library, we can pair up about that later.
I guess, for now, we can focus only on `manual-init` and let the `automatic init` for later.